src/Resources/skeletons/BaseExtension/bootstrap_package/composer.json.twig

```diff
@@ -7,9 +7,9 @@
     "keywords": ["TYPO3 CMS"],
     "version": "1.0.0",
```

Comment:
I would prever to set this version to 0.0.1 or eventually remove it

Reply:
nimm raus